src/librustc_error_codes/error_codes.rs

```diff
@@ -0,0 +1,13613 @@
+// Error messages for EXXXX errors.  Each message should start and end with a
+// new line, and be wrapped to 80 characters.  In vim you can `:set tw=80` and
+// use `gq` to wrap paragraphs. Use `:set tw=0` to disable.
+
+crate::register_diagnostics! {
+
+E0001: r##"
```

Comment:
So since you are doing this in an automated fashion, could we take the opportunity now to move all of these into `$code.md` files and just `include!(...)` their texts? Ideally we do this now to reduce the diffs.

Reply:
Do you mean having a markdown file for each of the error codes?

Reply:
Yes, exactly :)

Reply:
Ok, let's go for it then.